src/editor/js/main.js

```diff
@@ -1,29 +1,36 @@
-const canvas = document.getElementById('canvas');
-const ctx = canvas.getContext('2d');
+import {Editor} from './editor';
 
-const CANVAS_SIZE = canvas.clientWidth;
-const CELLS_COUNT = 16;
-const CELLS_STEP = CANVAS_SIZE / CELLS_COUNT;
+const editorContainer = document.querySelector('.editor .editor-wrapper');
+const rowsInput = document.querySelector('.editor .tools-rows');
+const colsInput = document.querySelector('.editor .tools-cols');
+const cellWidthInput = document.querySelector('.editor .tools-cell-width');
+const colourInput = document.querySelector('.editor .tools-colour');
+const resetButton = document.querySelector('.editor .tools-reset');
 
-const STROKE_WIDTH = 1;
-const STROKE_COLOR = '#000000';
+const editor = new Editor();
 
-canvas.width = CANVAS_SIZE;
-canvas.height = CANVAS_SIZE;
+const build = () => {
```

Comment:
This somewhat looks like duplicating logic of Editor class.
Maybe move this functionality into Editor?

I think nice apporach is to send container element to Editor constructor as a parameter. Then Editor can attach himself to provided element